integrationtests/src/test/java/org/axonframework/integrationtests/testsuite/administration/AnnotationTestDefinitions.java

```diff
@@ -0,0 +1,314 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.integrationtests.testsuite.administration;
+
+import jakarta.annotation.Nonnull;
+import org.axonframework.commandhandling.CommandMessage;
+import org.axonframework.commandhandling.CommandResultMessage;
+import org.axonframework.commandhandling.GenericCommandResultMessage;
+import org.axonframework.common.AxonConfigurationException;
+import org.axonframework.common.ReflectionUtils;
+import org.axonframework.common.annotation.AnnotationUtils;
+import org.axonframework.common.infra.ComponentDescriptor;
+import org.axonframework.common.infra.DescribableComponent;
+import org.axonframework.common.property.Property;
+import org.axonframework.eventhandling.EventMessage;
+import org.axonframework.eventsourcing.AnnotationBasedEventSourcedComponent;
+import org.axonframework.messaging.Message;
+import org.axonframework.messaging.MessageStream;
+import org.axonframework.messaging.QualifiedName;
+import org.axonframework.messaging.annotation.AnnotatedHandlerInspector;
+import org.axonframework.messaging.annotation.ParameterResolverFactory;
+import org.axonframework.messaging.unitofwork.ProcessingContext;
+import org.axonframework.modelling.command.EntityId;
+import org.axonframework.modelling.entity.EntityModel;
+import org.axonframework.modelling.entity.EntityModelBuilder;
+import org.axonframework.modelling.entity.PolymorphicEntityModel;
+import org.axonframework.modelling.entity.PolymorphicEntityModelBuilder;
+import org.axonframework.modelling.entity.child.ChildEntityFieldDefinition;
+import org.axonframework.modelling.entity.child.ListEntityChildModel;
+import org.axonframework.modelling.entity.child.SingleEntityChildModel;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.lang.reflect.AnnotatedElement;
+import java.lang.reflect.Field;
+import java.lang.reflect.Member;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import static java.lang.String.format;
+import static org.axonframework.common.ReflectionUtils.resolveMemberGenericType;
+import static org.axonframework.common.property.PropertyAccessStrategy.getProperty;
+
+public class AnnotationTestDefinitions {
```

Comment:
It's fair to assume I can disregard this class for review entirely for the time being, right?